src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    display: block;
+    margin-top: 4px;
+
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
it's a default value for this style, take a look at [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)